class UserException(Exception):
    pass

class DuplicateUserException(UserException):
    pass